operations/patch_module.go

```diff
@@ -177,6 +95,68 @@ func PatchSetModule() cli.Command {
 	}
 }
 
+func addModuleToPatch(params *patchParams, args cli.Args, conf *ClientSettings,
+	p *patch.Patch, module *model.Module, modulePath string) error {
+	patchId := p.Id.Hex()
+
+	preserveCommits := params.PreserveCommits || conf.PreserveCommits
+	if !params.SkipConfirm {
+		keepGoing, err := confirmUncommittedChanges(modulePath, preserveCommits, params.Uncommitted || conf.UncommittedChanges)
+		if err != nil {
+			return errors.Wrap(err, "confirming uncommitted changes")
+		}
+		if !keepGoing {
+			return errors.New("patch aborted")
+		}
+	}
+
+	ref := params.Ref
+	if params.Uncommitted || conf.UncommittedChanges {
+		ref = ""
+	}
+
+	// Diff against the module branch.
+	diffData, err := loadGitData(modulePath, module.Branch, ref, "", preserveCommits, args...)
+	if err != nil {
+		return err
+	}
+	if err = validatePatchSize(diffData, params.Large); err != nil {
+		return err
+	}
+
+	if len(diffData.fullPatch) == 0 {
+		grip.Infof("No changes for module '%s', continuing.", module.Name)
+		return nil
+	}
+	if !params.SkipConfirm {
+		grip.Infof("Using branch '%s' for module '%s'.", module.Branch, module.Name)
+		if diffData.patchSummary != "" {
+			fmt.Println(diffData.patchSummary)
+		}
+
+		if !confirm("This is a summary of the module patch to be submitted. Continue? (Y/n):", true) {
```

Comment:
Can we add a message here to confirm that we are skipping the module patch if the user prompts "no"?

Reply:
That feels a little unconventional since we don't typically respond to prompts i think but i tweaked the wording of the confirm message to be a bit clearer